app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -343,6 +343,22 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_loadExp_secondState_submitAnswer_clickSubmit_isToolbarTitleSelected() {
+    launchForExploration(TEST_EXPLORATION_ID_2).use {
+      startPlayingExploration()
+      clickContinueInteractionButton()
+      typeFractionText("1/2")
+
+      clickSubmitAnswerButton()
+
+      scrollToViewType(CONTINUE_NAVIGATION_BUTTON)
+      onView(withId(R.id.exploration_toolbar_title)).check(
+        matches(ViewMatchers.isSelected())
```

Comment:
Always try to use the end method import. 
In this cases not this `import androidx.test.espresso.matcher.ViewMatchers`, it should be `import androidx.test.espresso.matcher.ViewMatchers.isSelected`

Reply:
done